--- basicproperty-0.6.9a/basictypes/list_types.py   2004-05-11
16:32:37.000000000 +0530
+++ list_types.py  2009-04-28 22:58:46.000000000 +0530
@@ -213,37 +213,38 @@


 listof_strings = listof(
-   name = "listof_strings",
-   baseType = basic_types.String_DT,
-   dataType = "list.str",
+   basic_types.String_DT,
+   "listof_strings",
+   (),
 )
 listof_ints = listof(
-   name = "listof_ints",
-   baseType = basic_types.Int_DT,
-   dataType = "list.int",
+   basic_types.Int_DT,
+   "listof_ints",
+   (),
 )
 listof_bools = listof(
-   name = "listof_bools",
-   baseType = basic_types.Boolean_DT,
-   dataType = "list.bool",
+   basic_types.Boolean_DT,
+   "listof_bools",
+   (),
 )
 listof_longs = listof(
-   name = "listof_longs",
-   baseType = basic_types.Long_DT,
-   dataType = "list.long",
+   basic_types.Long_DT,
+   "listof_longs",
+   (),
 )
 listof_floats = listof(
-   name = "listof_floats",
-   baseType = basic_types.Float_DT,
-   dataType = "list.float",
+   basic_types.Float_DT,
+   "listof_floats",
+   (),
 )
 listof_classnames = listof(
-   name = "listof_classnames",
-   baseType = basic_types.ClassName_DT,
-   dataType = "list.str.classname",
+   basic_types.ClassName_DT,
+   "listof_classnames",
+   (),
 )
 listof_classes = listof(
-   name = "listof_classes",
-   baseType = basic_types.Class_DT,
-   dataType = "list.class",
+   basic_types.Class_DT,
+   "listof_classes",
+   (),
 )
 
